apps/demos/Demos/HtmlEditor/MarkdownSupport/jQuery/styles.css

```diff
@@ -3,12 +3,19 @@
   padding-right: 10px;
 }
 
+.value-title {
+  font-size: 18px;
+  font-weight: 500;
+}
+
 .value-content {
   margin-top: 20px;
   overflow: auto;
-  height: 110px;
-  width: 100%;
+  height: 210px;
   white-space: pre-wrap;
+  border: 1px solid #808080;
```

Comment:
We need to check all themes and color modes (including dark mode) to ensure everything looks good. For me, it's better to use CSS variables here, e.g., --dx-color-border. 
Kindly approve the final view with designer.